lnrpc/walletrpc/walletkit_server.go

```diff
@@ -985,11 +985,25 @@ func (w *WalletKit) BumpFee(ctx context.Context,
 		Force: in.Force,
 	}
 
-	_, err = w.cfg.Sweeper.UpdateParams(*op, params)
+	sweepChan, err := w.cfg.Sweeper.UpdateParams(*op, params)
 	switch err {
 	case nil:
+		var sweepTxHex string
+		if in.IncludeRawTx {
+			select {
+			case sweepResult := <-sweepChan:
```

Comment:
To make sure the user can cancel this operation (and to also fix the linter issue), we must listen on `ctx.Done()` here which fires when the user aborts on the client side (Ctrl+C). Same below.